src/driver-hidpp10.c

```diff
@@ -656,7 +656,15 @@ hidpp10drv_probe(struct ratbag_device *device)
 		else if (strcasecmp("G9", typestr) == 0)
 			type = HIDPP10_PROFILE_G9;
 
-		profile_count = ratbag_device_data_hidpp10_get_profile_count(device->data);
+		rc = ratbag_device_data_hidpp10_get_profile_count(device->data);
+		if (rc == -1) {
+			log_error(device->ratbag,
+				  "Device %s has no profile count set, even though profiles are enabled. "
+				  "Please adjust the .device file.\n",
+				  device->name);
```

Comment:
Should it fall back to `profile_count = 1` here?

Reply:
It's initialized with `1` above
https://github.com/libratbag/libratbag/blob/931ec44ff27fd2ef78b830bbe20acd2e26576a9f/src/driver-hidpp10.c#L639